src/index.js

```diff
@@ -118,43 +144,40 @@ class IOpipeWrapperClass {
       s: stringValue
     });
   }
-  invoke(originalEvent, originalContext, originalCallback) {
-    this.invokeSetup(originalEvent, originalContext, originalCallback);
-    try {
-      return this.userFunc.call(
-        this,
-        this.originalEvent,
-        this.modifiedContext,
-        this.modifiedCallback
-      );
-    } catch (err) {
-      return this.sendReport(err);
-    }
-  }
 }
 
 module.exports = options => {
   const config = setConfig(options);
-  const fn = userFunc => {
+  const dnsPromise = getDnsPromise(config.host);
+  const libFn = userFunc => {
     if (!config.clientId) {
       // No-op if user doesn't set an IOpipe token.
       return userFunc;
     }
 
-    const dnsPromise = getDnsPromise(options.host);
+    // Assign .log (deprecated) here to avoid type errors
+    if (typeof libFn.log !== 'function') {
```

Comment:
I guess the idea is so that if someone does:

```
var iopipe = require('iopipe')()

iopipe.log('metric', 1) // not called during an invocation
```

that it will do a no-op instead of an error... this does break an existing contract. Not sure it's one we should keep or one that anyone's using, but it is breaking.